.github/workflows/dependabot.adhoc.yml

```diff
@@ -0,0 +1,26 @@
+name: Dependabot Batcher Ad-hoc
+on: workflow_dispatch
+
+jobs:
+  # Batches Dependabot PRs into one by merging them into a combined branch, then raising a new PR
+  dependabot-batcher:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+        with:
+          token: ${{ secrets.GITHUB_TOKEN }}
+
+      - uses: actions/setup-node@v2.5.1
+        with:
+          node-version-file: '.nvmrc'
+
+      - name: 'Combine Dependabot PRs'
+        uses: actions/github-script@v5
+        with:
+          github-token: ${{ secrets.GITHUB_TOKEN }}
+          script: |
+            const combineDependabotPrs = require('./.github/workflow-scripts/combine-dependabot-prs.js');
+            return await combineDependabotPrs({
+              github,
+              context,
+            });
```

Comment:
Am I right by saying this job is the same as in the other two workflow riles? I'm not super familiar with Github workflows, is there a way to store a job and then call on it inside multiple workflows instead of repeating it?

Reply:
This does the same thing as `dependabot.scheduled.yml`. The only difference is that their run is triggered by different events: one manually bu a user (adhoc) and the other one on schedule.

There is a way of reusing workflows but we still need to look into it as it's a fairly new feature. 
We will be refactoring this code once we have time to investigate the above 👍

Reply:
Thanks, sounds good 👍